### ISSUE TITLE
Update marketplace (MP) docs page to cover switching from direct to MP subscription

### ISSUE THREAD

**Issue by aashishkohli** [ClickHouse/clickhouse-docs#2075]
Currently we describe the steps for MP signup as https://clickhouse.com/docs/knowledgebase/marketplace#how-can-i-start-using-clickhouse-cloud-connected-to-my-awsgcpazure-account-billing

For customers that are already subscribed to CH Cloud but then want the billing to happen through MP for the same org, we have this section: https://clickhouse.com/docs/knowledgebase/marketplace#i-am-an-existing-clickhouse-cloud-user-and-i-want-my-existing-services-to-be-billed-via-marketplace

However, is it not clear from the first section whether a user needs to take additional steps to connect an existing CH Cloud org to a MP subscription. We need to clarify this. I will provide updated sections.

**Comment by aashishkohli** [comment]
https://clickhouse-inc.slack.com/archives/C04ASGUS7TJ/p1708714635850119

**Comment by aashishkohli** [comment]
@justindeguzman - I will provide a google doc with updated marketplace docs. We should just replace the current docs page with the updated one.

**Comment by gingerwizard** [comment]
@aashishkohli is this done?

**Comment by aashishkohli** [comment]
Yes @gingerwizard. We split this into 6 different pages - 2 for each CSP. I'm closing this issue.
